monitor/cmd/monitor-server/server.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/mesosphere/performance/api/bigquery"
```

Comment:
I was hoping this component could be written in a way that it wasn't tied to bigquery (and just sent JSON to the api server). That way we can move to a different Storage backend in the future and only have to change the API server and not all of our various monitors/tests.